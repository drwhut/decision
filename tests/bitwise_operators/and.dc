Start~#1

> false AND false = false
And(false, false)~#2
Print(#1, #2)~#3

> false AND true = false
And(false, true)~#4
Print(#3, #4)~#5

> true AND false = false
And(true, false)~#6
Print(#5, #6)~#7

> true AND true = true
And(true, true)~#8
Print(#7, #8)~#9