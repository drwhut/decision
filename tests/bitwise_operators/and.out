false
false
false
true
1216
